org.openscad.OpenSCAD.json

```diff
@@ -0,0 +1,164 @@
+                    "sha256": "0454b6bacafd2bf641e0fb0f59572b9995728b77e41d1b8517f807334d07e68e"
+                }
+            ]
+        },
+        {
+            "name": "cgal",
+            "cleanup": [
+                "/bin"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-4.11.2/CGAL-4.11.2.tar.xz",
+                    "sha256": "a848f62968f545d46fd9b38f513983c04d3cc22eb59564ed4b3866d8626e7551"
+                }
+            ]
+        },
+        {
+            "name": "opencsg",
+            "no-autogen": true,
+            "no-parallel-make": true,
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.opencsg.org/OpenCSG-1.4.2.tar.gz",
+                    "sha256": "d952ec5d3a2e46a30019c210963fcddff66813efc9c29603b72f9553adff4afb"
+                },
+                {
+                    "type": "patch",
+                    "path": "opencsg-libs.patch"
+                }
+            ]
+        },
+        {
+            "name": "qscintilla",
+            "buildsystem": "qmake",
+            "subdir": "Qt4Qt5",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/pyqt/files/QScintilla2/QScintilla-2.10.3/QScintilla_gpl-2.10.3.tar.gz",
+                    "sha256": "d2657a68e7699d37209ff97db965de7c67884d99b33066d0ba457e6f39b581e7"
+                },
+                {
+                    "type": "patch",
+                    "path": "qscintilla-lib-paths.patch"
+                }
+            ]
+        },
+        {
+            "name": "openscad",
+            "buildsystem": "qmake",
+            "config-opts": [
+                "QMAKE_CFLAGS_ISYSTEM=",
+                "QMAKE_LIBDIR=/app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://files.openscad.org/openscad-2015.03-2.src.tar.gz",
+                    "sha256": "a2535bb9e27d96e10e1e19268a596155164c7129d410a0c7f96edc2d09400083"
+                },
+                {
+                    "type": "patch",
+                    "path": "UIC-gettext-interoperability.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "openscad-libscintilla.patch"
+                }
+            ],
+            "post-install": [
+                "mkdir -p /app/share/icons/hicolor/512x512/apps",
+                "mv /app/share/pixmaps/openscad.png /app/share/icons/hicolor/512x512/apps",
+                "mv /app/share/mime/packages/openscad.xml /app/share/mime/packages/org.openscad.OpenSCAD.xml"
```

Comment:
I've not looked at the contents but usually you need to `sed` the icon name with the correct one.

Reply:
Ah yeah... reminds me of https://github.com/flatpak/flatpak-builder/issues/1 !